build/server/host/Host.tsx

```diff
@@ -31,13 +27,16 @@ function HostWrapper() {
 
   return (
     <AppProvider theme={theme} i18n={{}}>
-      <Host script={reactThirdPartyWorker.url} />
+      <Host script={THIRD_PARTY_SCRIPT} />
     </AppProvider>
   );
 }
 
 function Host(props: HostProps) {
-  switch (config.EXTENSION_TYPE) {
+  const config = {
```

Comment:
Why do we hardcode this?

Reply:
Good catch, I forgot to add this as an arg in `server`. Let me add that.